refinery/core/api.py

```diff
@@ -1852,7 +1853,9 @@ def get_member_list(self, ext_group):
                 'last_name': u.last_name,
                 'is_manager': self.user_authorized(u, ext_group)
             },
-            ext_group.user_set.all().filter(is_active=True))
+            ext_group.user_set.all().exclude(Q(is_active=False) |
```

Comment:
https://docs.djangoproject.com/en/2.0/ref/models/querysets/#methods-that-return-new-querysets

Reply:
"If you need to execute more complex queries (for example, queries with OR statements), you can use Q objects."

Reply:
I think you could avoid the use of `Q()` objects with a query like: `ext_group.user_set.all().filter(is_active=True).exclude(id=settings.ANONYMOUS_USER_ID)`

They're not bad to use, but I've generally only seen them used when the standard Django ORM falls short.

Reply:
@scottx611x What advantage is there to chaining filter & exclude vs Q? Skimming the docs it appears Q would work well for this OR situation.